Subsystem/AttributeLoader.cs

```diff
@@ -488,8 +489,7 @@ public void ApplyWeaponAttributesPatch(WeaponAttributesPatch weaponAttributesPat
             {
                 using (logger.BeginScope($"Turret:"))
                 {
-                    var turret = weaponAttributesWrapper.Turret;
-                    var turretWrapper = new TurretAttributesWrapper(turret);
+                    var turretWrapper = new TurretAttributesWrapper(weaponAttributesWrapper.Turret);
```

Comment:
General PR hygiene: Avoid making changes like this to unrelated code.

(Going to try this "suggestions" thing...)

```suggestion
                    var turret = weaponAttributesWrapper.Turret;
                    var turretWrapper = new TurretAttributesWrapper(turret);
```